### DISCUSSION TITLE
Output expressions delay (timeout).

### DISCUSSION THREAD

**Discussion by pavvd** [houmain/keymapper#58] [Ideas]
First I have to say that this tool is amazing. Not only it works better than anything I tried so far on Linux, but the config is so elegant and simple.

So are there any plans on adding delay parameter to Output Expressions? Something like:
`Meta{C} >> {200ms}Meta{R} ^ C M D Enter`
That would cause 200ms break between sending next key in sequence. 
Currently l'm using xdotool --delay inside terminal binding $( ), it works most of the time.

**Comment by houmain** [comment]
Thanks! I need to start a list of testimonial quotes...

Good idea, I will try to implement it in the next few weeks.
